### DISCUSSION TITLE
Skip support of Text.fontWidth() [iOS 16]

### DISCUSSION THREAD

**Discussion by davidakoontz** [skiptools/skip#184] [Q&A] [answered]
Well I'm a newbie with SKIP.tools and porting a newly being built App to Android seems like Skip is the way to go.

My first issue was with WindowGroup in the App.  But it appears the ScrumSkipper example answers this issue nicely.

My next set of issues were with Text .fontWidth() [iOS 16].  I'm using condensed fonts.  But Skip doesn't support this feature yet...
I have no idea how much effort there is in supporting fonts...  but seems this is going to be a bigger issue in the near future.

https://developer.apple.com/documentation/SwiftUI/Font/Width

Getting-standard-font-widths:
    static let compressed: Font.Width
    static let condensed: Font.Width
    static let expanded: Font.Width
    static let standard: Font.Width

Any known good workarounds for fontWidth()?

**Comment by marcprux** [comment, marked as answer]
There doesn't appear to be any standard Jetpack Compose modifier for specifying the font width, so you might have to bundle a custom font yourself to get this effect on the Android side: https://skip.tools/docs/modules/skip-ui/#custom-fonts.

**Reply by davidakoontz** [reply]
Oh ... no...  Hey thanks... in this case, I'm OK with regular font widths...

No - this guy is not going down that rabbit hole.

I'm excited to just have a working Android App... font issues can be ignored - I'm a developer... not a designer.
